lib/readline.js

```diff
@@ -929,17 +932,27 @@ function emitKeypressEvents(stream, iface) {
   stream[ESCAPE_DECODER] = emitKeys(stream);
   stream[ESCAPE_DECODER].next();
 
+  const escapeCodeTimeout = () => stream[ESCAPE_DECODER].next('');
+
   function onData(b) {
     if (stream.listenerCount('keypress') > 0) {
       var r = stream[KEYPRESS_DECODER].write(b);
       if (r) {
+        if (timeoutRef) {
+          timeoutRef = clearTimeout(timeoutRef);
```

Comment:
@Fishrock123 Yes, I just assigned the `undefined` return value to timeout object. I'll remove that